deploy/templates/nstemplatetiers/test/ns_dev.yaml

```diff
@@ -13,64 +13,6 @@ objects:
     labels:
       name: ${USERNAME}-dev
     name: ${USERNAME}-dev
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: Role
-  metadata:
-    name: rbac-edit
-    namespace: ${USERNAME}-dev
-  rules:
-    - apiGroups:
-        - authorization.openshift.io
-        - rbac.authorization.k8s.io
-      resources:
-        - roles
-        - rolebindings
-      verbs:
-        - get
-        - list
-        - watch
-        - create
-        - update
-        - patch
-        - delete
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: RoleBinding
-  metadata:
-    name: user-rbac-edit
-    namespace: ${USERNAME}-dev
-  roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: Role
-    name: rbac-edit
-  subjects:
-    - kind: User
-      name: ${USERNAME}
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: RoleBinding
-  metadata:
-    name: user-edit
-    namespace: ${USERNAME}-dev
-  roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: ClusterRole
-    name: edit
-  subjects:
-  - kind: User
-    name: ${USERNAME}
-- apiVersion: v1
-  kind: LimitRange
-  metadata:
-    name: resource-limits
-    namespace: ${USERNAME}-dev
-  spec:
-    limits:
-    - type: "Container"
-      default:
-        cpu: 1000m
-        memory: 750Mi
-      defaultRequest:
-        cpu: 10m
-        memory: 64Mi
```

Comment:
why did you remove this LimitRange?

Reply:
definitely NOT intended! I'll restore that

Reply:
restored in 6947622